types/foundry/client-esm/applications/_types.d.ts

```diff
@@ -39,7 +39,7 @@ export interface ApplicationWindowConfiguration {
     icon?: string | false;
 
     /** An array of window control entries */
-    controls: ApplicationHeaderControlsEntry[];
+    controls?: ApplicationHeaderControlsEntry[];
```

Comment:
```suggestion
    controls: ApplicationHeaderControlsEntry[];
```
Keep this required since it's always present from the defaults.